change/@office-iss-react-native-win32-2021-01-08-12-25-41-sammy-0.63-stable.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Comment:
This needs to be changed to "patch" instead of "none" to generate a new release

Reply:
Thanks, changing now